addOns/graphql/src/main/java/org/zaproxy/addon/graphql/Requestor.java

```diff
@@ -49,6 +55,61 @@ public Requestor(int initiator) {
                 HttpRequestConfig.builder().setRedirectionValidator(new MessageHandler()).build();
     }
 
+    public HttpMessage sendQueryByGet(String query) {
+        HttpMessage message = null;
+        try {
+            URI url =
+                    UrlBuilder.build(
+                            endpointUrl
+                                    + "?query="
+                                    + URLEncoder.encode(query, StandardCharsets.UTF_8.toString()));
+            message = new HttpMessage(url);
+            send(message);
+        } catch (IOException e) {
+            LOG.error(e.getMessage());
+        }
+        return message;
+    }
+
+    public HttpMessage sendQueryByGraphQlPost(String query) {
+        HttpMessage message = null;
+        try {
+            HttpRequestBody msgBody = new HttpRequestBody(query);
+            HttpRequestHeader msgHeader =
+                    new HttpRequestHeader(HttpRequestHeader.POST, endpointUrl, "HTTP/1.1");
+            msgHeader.setHeader("Accept", "application/json");
+            msgHeader.setHeader("Content-Type", "application/graphql");
```

Comment:
There are probably core constants for a bunch of these, check `HttpHeader` etc. (Maybe not the content types, but the HTTP declaration and header field names I'm pretty sure about). Maybe make local constants for the content types.